pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -70,8 +71,10 @@ const (
 	// were to continuously return a nextPageToken.
 	maxPages = 25
 
-	// Target Pool creation is limited to 200 instances.
 	maxTargetPoolCreateInstances = 200
+
+	// Max size of names accepted by the GCE api.
+	nameLengthLimit = 63
```

Comment:
discussed IRL - this should be the same name as the FwdRule and TargetPool, I think.